devops/nightly-perf.yml

```diff
@@ -4,6 +4,9 @@ trigger: none # No CI build
 
 pr: none # Not for pull requests
 
+variables:
+- group: "Key Vault"
```

Comment:
I would be interested in how this is working. It would not be hard to concoct something better than what I did for the releases, and we should standardise

Reply:
Sure, let's discuss in person!

Reply:
Outcome from discussion: I will change this to using the keyvault task, that way everything is in the YAML as opposed to obscure ADO definitions of things.